src/TextEditor.js

```diff
@@ -7,9 +7,9 @@
 import React, {Component, PropTypes} from 'react';
 import classNames from 'classnames';
 import Immutable from 'immutable';
-import { Editor, EditorState, ContentState, convertFromHTML, RichUtils, convertToRaw, convertFromRaw, CompositeDecorator } from 'draft-js';
+import { Editor, EditorState, ContentState, RichUtils, convertToRaw, convertFromRaw, CompositeDecorator, Modifier } from 'draft-js';
 import { stateToHTML } from 'draft-js-export-html';
-
+import {convertFromHTML} from 'draft-convert';
```

Comment:
We are using two libraries that do the same thing at the moment. Let's get rid of `draft-js-export-html` and just use `draft-convert`